app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    listed_command = command.split()
```

Comment:
add if len(command.split()) == 3:, because if you have less arguments your code will failed